src/hooks/useFetchForwardUrls.tsx

```diff
@@ -0,0 +1,16 @@
+import { gql } from '@apollo/client';
+
+export const FORWARD_URL_FRAGMENT = gql`
+  fragment ForwardUrl on ForwardUrl {
+    url
+  }
+`;
+
+export const GET_FORWARD_URLS = gql`
+  query GetForwardUrls($endpointId: ID!) {
+    forwardUrls(endpointId: $endpointId) {
+      ...ForwardUrl
+    }
+  }
+  ${FORWARD_URL_FRAGMENT}
+`;
```

Comment:
* All queries have been moved to `src/schema/queries.ts` and fragments to `src/schema/fragments.ts`, so these pieces should move there as well
* As an FYI, there's nothing `hook` like in this file, so the filename shouldn't start with `use`

Reply:
Looks like the queries and fragments are already there. deleted file.